### ISSUE TITLE
Improve translations

### ISSUE THREAD

**Issue by NathanBnm** [phase1geo/Minder#69]
Hi, I figured out about something when doing #68. I noticed that you added some HTML tags in your app to make some text **bold** or *italic* for example, but this hasn't to be in the translations.

You have to use concatenation to solve this. Here is an example with `MapInspector.vala`:

```diff
- var lbl     = new Label( _( "<b>Enable animations</b>" ) );
+ var lbl     = new Label("<b>" +  _( "Enable animations" )  + "</b>");
```

```diff
#: src/MapInspector.vala:61
- msgid "<b>Enable animations</b>"
- msgstr "<b>Activer les animations</b>"
+ msgid "Enable animations"
+ msgstr "Activer les animations"
```

**Note**: In your file `EmptyInspector.vala` you're using the `<big>` HTML tag but you should not use it as it is deprecated. See [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/big) for further information.

**Comment by phase1geo** [comment]
I have removed all of the markup from translatable strings in the 'focus' development branch. This code change will be generally available in the 1.4 upcoming release.

As for the usage of <big>, it should be allowable in the context that I am using it in. This markup is allowable for Pango text markup as described in their documentation. I don't see any reference to this markup being obsoleted in the Pango text markup documentation. This markup isn't required to follow the HTML standard.

**Comment by phase1geo** [comment]
Closing bug now that 1.4.0 is in the release pipeline.
